src/components/interaction-designer/resource-editors/ResourceEditor.vue

```diff
@@ -72,40 +72,40 @@ import {
   Mutation,
   namespace,
 } from 'vuex-class'
-  import {
-    IBlock,
-    IFlow,
-    IResourceDefinition,
-    SupportedContentType,
-    SupportedMode,
-  } from '@floip/flow-runner'
-  import lang from '@/lib/filters/lang'
-  import Permissions from '@/lib/mixins/Permissions'
-  import Routes from '@/lib/mixins/Routes'
-  import FlowUploader from '@/lib/mixins/FlowUploader'
-  import {Component} from 'vue-property-decorator'
-  import Vue from 'vue'
-  import ResourceVariantTextEditor from './ResourceVariantTextEditor.vue'
-  import {
+import {
+  IBlock,
+  IFlow,
+  IResourceDefinition,
+  SupportedContentType,
+  SupportedMode,
+} from '@floip/flow-runner'
+import lang from '@/lib/filters/lang'
+import Permissions from '@/lib/mixins/Permissions'
+import Routes from '@/lib/mixins/Routes'
+import FlowUploader from '@/lib/mixins/FlowUploader'
+import { Component } from 'vue-property-decorator'
+import Vue from 'vue'
+import {
   discoverContentTypesFor,
-    findOrGenerateStubbedVariantOn,
-    findResourceVariantOverModesOn
-  } from '@/store/flow/resource'
-  import AudioLibrarySelector from '@/components/common/AudioLibrarySelector.vue'
-  import UploadMonitor from '../block-editors/UploadMonitor.vue'
-import ValidationException from "@floip/flow-runner/src/domain/exceptions/ValidationException";
-import {cloneDeep} from "lodash";
+  findOrGenerateStubbedVariantOn,
+  findResourceVariantOverModesOn,
+} from '@/store/flow/resource'
+import AudioLibrarySelector from '@/components/common/AudioLibrarySelector.vue'
+import ValidationException from '@floip/flow-runner/src/domain/exceptions/ValidationException'
+import { cloneDeep } from 'lodash'
+import UploadMonitor from '../block-editors/UploadMonitor.vue'
+import ResourceVariantTextEditor from './ResourceVariantTextEditor.vue'
 
-  const flowVuexNamespace = namespace('flow')
+const flowVuexNamespace = namespace('flow')
 
   interface IAudioFile {
-    id: string,
-    filename: string,
-    description: string,
-    language_id: string,
-    duration_seconds: string,
-    original_extension: string,
-    created_at: string
+    id: string;
```

Comment:
@bzabos How do you feel about semicolons here? I think our standard in flow-runner is commas?

https://github.com/FLOIP/flow-runner

Reply:
The semicolon here is added for the properties of an interface by ESLint. I am not sure where this rule is getting applied from. We may not be able to use commas in an interface but we can omit having a semicolon.

> Love it! Just a couple of things outstanding from my POV - line length and those questions for @bzabos on semicolons.
> 
> One larger thing though. I'm not sure we'll get all the value from this unless we enforce it going forward. Can we add it to the travis build as part of this ticket? What do you think @poojahpatil90, @bzabos, @safydy?

It makes sense to add it as part of travis. We could additionally try to enforce it by enabling lintOnSave to true in the vue.config.js file.

Reply:
Looks like I'm wrong here. Not sure what I was looking at in the flow-runner to make me think this. Looks like we are inconsistent there but it's mostly semicolons...